### DISCUSSION TITLE
Re Koneksi Device

### DISCUSSION THREAD

**Discussion by mediasaranainformatika** [kirimwa-id/forum#118] [Tanya Jawab]
Salam mas Rio,

Mohon maaf sebelumnya, jika ada pesan status diskoneksi seperti ini di response nya : 

"**_disconnected_reason": "Cannot set headers after they are sent to the client"_**

itu kendalanya dimana ya mas?

**Comment by rioastamal** [comment]
@mediasaranainformatika, apakah anda memasang webhook?

**Reply by mediasaranainformatika** [reply]
Tidak mas

**Reply by rioastamal** [reply]
Aneh sekali. Mungkin silahkan menunggu versi MD nanti, semoga bisa fixing issue tersebut.

**Comment by mediasaranainformatika** [comment]
Siap mas rio, semoga bs fix issue nya di versi berikutnya
